Packs/TrendMicroVisionOne/ModelingRules/TrendMicroVisionOneModelingRules/TrendMicroVisionOneModelingRules.xif

```diff
@@ -33,123 +33,150 @@ filter
     mitreTechniques = arraymap(Rules, replex(arraystring(json_extract_array(arrayindex(json_extract_array("@element", "$.matchedFilters"), 0), "$.mitreTechniqueIds"), ", "), "\"", ""))
 
     | alter
-    xdm.source.cloud.project = id,
-    xdm.event.outcome_reason = investigationStatus,
-    xdm.network.http.url = workbenchLink,
-    xdm.source.application.name = alertProvider,
-    xdm.event.type = model,
+    xdm.alert.original_threat_name = originalThreatName,
+    xdm.alert.mitre_techniques = mitreTechniques,
     xdm.alert.severity = to_string(score),
-    xdm.event.log_level = severity,
+    xdm.alert.status = if(status = "Open", XDM_CONST.ALERT_STATUS_PENDING, status = "In Progress", XDM_CONST.ALERT_STATUS_IN_REVIEW, XDM_CONST.ALERT_STATUS_DONE),
     xdm.alert.description = description,
     xdm.alert.original_alert_id = matchedRulesIds,
     xdm.alert.name = matchedRulesNames,
+    xdm.alert.subcategory = provenances,
     xdm.source.user.identity_type = identityType,
     xdm.source.user.username = coalesce(entityValuesUserName, entityValueGuidsUserName, entityValueNamesUserName),
     xdm.source.host.device_id = entityIdsDeviceId,
     xdm.source.user.identifier = coalesce(entityIdsAccountIdentifier, entityIdsEmailIdentifier),
     xdm.source.host.hostname = coalesce(entityValueGuidsHostname, entityValueNamesHostName, indicatorsValuesHostname),
     xdm.source.user.upn = coalesce(entityValueGuidsUpn, entityValueNamesUpn),
-    xdm.target.resource.id = relatedEntities,
-    xdm.alert.subcategory = provenances,
+    xdm.source.process.command_line = indicatorsValuesCommandLine,
+    xdm.source.process.executable.path = indicatorsValuesExecutablePath,
+    xdm.source.application.name = alertProvider,
     xdm.source.ipv4 = entityValueIps,
```

Comment:
Is it only ipv6 values ? If not, we need the value checks and adding the xdm.source.ipv6 field.